staging_service/autodetect/GenerateMappings.py

```diff
@@ -28,223 +28,114 @@
 
 from staging_service.autodetect.Mappings import *
 
-mapping = defaultdict(list)
+# Note that some upload apps are not included - in particular batch apps, which are now
+# redundant, and MSAs because they're out of scope at the current time and don't conform to
+# standards (what does this mean?).
```

Comment:
Maybe we should answer this question instead of putting in a question in the comment?

Reply:
I would like to, but I don't know the answer - I copied that line from the previous state of the file. Do you know what 'the MSA doesn't conform to standards' means?

Reply:
Don't remember. Maybe we just say its out of scope then.

Reply:
Done